ios/Classes/FullstoryFlutterPlugin.swift

```diff
@@ -164,6 +164,24 @@ public class FullstoryFlutterPlugin: NSObject, FlutterPlugin, FSDelegate {
             } else {
                 result(FlutterError(code: "INVALID_PAGE", message: "No active page found with the given ID", details: nil))
             }
+        case "captureEvent":
+            if let args = call.arguments as? [String: Any] {
+                if let FS = NSClassFromString("FS") {
+                    let sel = NSSelectorFromString("__flutterEvent:")
+                    if let clazz = FS as AnyObject as? NSObjectProtocol {
+                        if clazz.responds(to: sel) {
+                            let obj = clazz.perform(sel, with: args)
+                            if(obj != nil) {
+                                result(nil)
+                                return
+                            }
```

Comment:
This is another kind of error - unexpected return value. Otherwise, the captureEvent code LGTM. (We can always revisit if there is a perf issue with the dynamic message send.)

Reply:
Added another message for that case, thanks!